client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -601,8 +612,9 @@ class WpcomChecklistComponent extends PureComponent {
 			<TaskComponent
 				{ ...baseProps }
 				bannerImageSrc="/calypso/images/stats/tasks/mobile-app.svg"
-				completedButtonText={ translate( 'Download mobile app' ) }
+				completedButtonText={ translate( 'Re-download mobile app' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.